exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -101,6 +111,7 @@ func newPRWTelemetry(set exporter.Settings) (prwTelemetry, error) {
 		telemetryBuilder: telemetryBuilder,
 		otelAttrs: []attribute.KeyValue{
 			attribute.String("exporter", set.ID.String()),
+			attribute.String("endpoint", endpointURL.String()),
```

Comment:
If we add this attribute to otelAttrs, will this be added to all metrics of the exporter? Or just the new ones you're adding in the PR?

Reply:
Depends on what you mean with "all metrics of the exporter".

It'll certainly be added to all metrics defined specifically for the exporter but not the generic ones, in this case we'll add them to:

- `otelcol_exporter_prometheusremotewrite_consumers`
- `otelcol_exporter_prometheusremotewrite_sent_batch_count`
- `otelcol_exporter_prometheusremotewrite_translated_time_series`
- `otelcol_exporter_prometheusremotewrite_failed_translations`

but **not** to other metrics such as`otelcol_exporter_queue_capacity` or `otelcol_exporter_send_failed_metric_points`

Reply:
<img width="1436" alt="image" src="https://github.com/user-attachments/assets/b2208a4a-0ad8-4a0b-839b-ef7999b39c5e" />